recipes/samarium/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import functools
+
+required_conan_version = ">=1.43.0"
+
+
+class SamariumConan(ConanFile):
+    name = "samarium"
+    description = "2-D physics simulation library"
+    homepage = "https://strangequark1041.github.io/samarium/"
+    url = "https://github.com/conan-io/conan-center-index/"
+    license = "MIT"
+    topics = ("c++20", "physics", "2d", "simulation")
```

Comment:
```suggestion
    topics = ("cpp20", "physics", "2d", "simulation")
```
consider topics to be searched in a web page, where both white space and plus signal means, search for two different words